### PR TITLE
Added text and deleted

### PR DESCRIPTION
**Pull request by diamonddocumentation**

### CODE REVIEW

File: test-homepage [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. 
+
 Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend 
 ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque 
 rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas 
 tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit vel, 
 luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam 
 quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. 
 Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,
+
```
Comment (oscarisalittledog) [inline test-homepage:12]: This is a comment on the file

File: test-homepage [hunk 2]
```diff
@@ -1,11 +1,13 @@
 Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et 
 magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat 
 massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis 
-vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. 
+
 Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend 
 ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque 
 rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas
```
Comment (oscarisalittledog) [inline test-homepage:7]: This line is a bit funny

File: test-homepage [hunk 3]
```diff
@@ -1,11 +1,13 @@
 Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et 
 magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat 
 massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis 
-vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. 
+
 Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend 
 ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque
```
Comment (oscarisalittledog) [inline test-homepage:6]: change this to bold

**Review by oscarisalittledog — COMMENTED** [review]
I have made some comments

File: test-homepage (hunk 3 above)
Comment (oscarisalittledog) [reply test-homepage:6]: ok

Comment (diamonddocumentation) [reply test-homepage:6]: ok

File: test-homepage (hunk 2 above)
Comment (diamonddocumentation) [reply test-homepage:7]: I will rewrite it

File: test-homepage (hunk 1 above)
Comment (diamonddocumentation) [reply test-homepage:12]: ok i will revise  it

**Review by oscarisalittledog — COMMENTED** [review]
I have left some comments

File: test-homepage (hunk 3 above)
Comment (diamonddocumentation) [reply test-homepage:6]: It is 18.18 add this

**Review by diamonddocumentation — COMMENTED** [review]
I'm not sure about this. I have changed it to 18.23

**Comment by oscarisalittledog** [comment]
All changes merged

**Pull request merged**